dmoj/settings.py

```diff
@@ -110,6 +110,10 @@
 DMOJ_PASSWORD_RESET_LIMIT_WINDOW = 3600
 DMOJ_PASSWORD_RESET_LIMIT_COUNT = 10
 
+DMOJ_EMAIL_CHANGE_LIMIT_WINDOW = 3600
```

Comment:
Might want to add comments for units / explaining purpose (my first thought was `DMOJ_EMAIL_CHANGE_LIMIT_COUNT` was a hard cap on the number of times you could change your email in total.